demo/llm.chatui.service/simple-chat.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
   namespace: ${MODEL_NAMESPACE}
```

Comment:
Quick question about this: In llm.rag.service/chat-serveragllmpluslb.yaml and in llm.vdb.service/createvdb.yaml, we do not use the MODEL_NAMESPACE and instead just use default. So this namespace is not specified in our install docs here so far: https://github.com/elotl/GenAI-infra-stack/blob/main/docs/install.md So we could either (1) add MODEL_NAMESPACE to the other manifests and include in the install docs or (2) remove from here so we can use the same namespace for all three components. What do you think?

Reply:
let's discuss what we prefer and I will do it in a separate PR.